apps/fabric-tester/ios/fabrictester.xcodeproj/project.pbxproj

```diff
@@ -355,8 +355,18 @@
 				);
 				INFOPLIST_FILE = fabrictester/Info.plist;
 				IPHONEOS_DEPLOYMENT_TARGET = 13.0;
-				LD_RUNPATH_SEARCH_PATHS = "$(inherited) @executable_path/Frameworks";
+				LD_RUNPATH_SEARCH_PATHS = (
+					"$(inherited)",
+					"@executable_path/Frameworks",
+				);
 				MARKETING_VERSION = 1.0;
+				OTHER_CPLUSPLUSFLAGS = (
+					"$(inherited)",
+					"-DRCT_NEW_ARCH_ENABLED=1",
+					"-DFOLLY_NO_CONFIG",
+					"-DFOLLY_MOBILE=1",
+					"-DFOLLY_USE_LIBCPP=1",
+				);
```

Comment:
are the changes intentional? does user need to change these in their project?

Reply:
Ops, these changes are not necessary, I've just reverted it